index.html

```diff
@@ -290,5 +304,6 @@ <h2 class="contact-title" >Contact me</h2>
         </form>
       </fieldset>
     </section>
```

Comment:
- [ ] The language section is overlapping the contact section on mobile view. Please kindly fix that.

Reply:
Which part overlaps?

![Screen Shot 2023-01-03 at 15 06 57](https://user-images.githubusercontent.com/44959066/210354331-eae71b7d-ba57-4cfb-8fd5-ed3b2d445671.png)

Reply:
![two](https://user-images.githubusercontent.com/61285247/210359594-036cf908-3d57-4090-969b-2bf4f35cda5a.png)
This is the overlapping am talking about.

Reply:
Fixed it. Thanks

![Screen Shot 2023-01-03 at 17 17 09](https://user-images.githubusercontent.com/44959066/210374989-d36a5363-8168-4a3d-ae2e-9682d0c20b95.png)